src/SFA.DAS.AssessorService.Web.UnitTests/CertificateTests/CertificateVersionControllerTests.cs

```diff
@@ -117,13 +136,13 @@ public async Task WhenSelectingAVersion_WhenLoadingModel_WithOneVersion_Redirect
         }
 
         [Test, MoqAutoData]
-        public async Task WhenSelectingAVersion_WhenLoadingModel_WithOneVersion_RedirectsToOptionPage(CertificateSession session, StandardVersion standardVersion, StandardOptions options)
+        public async Task WhenSelectingAVersion_WhenLoadingModel_WithOneVersion_RedirectsToOptionPage(CertificateSession session, SharedModelTypes.StandardVersion standardVersion, StandardOptions options)
```

Comment:
Not immediately obvious from the name of this test that it is different from the one above. Might be worth including `_WithOptions`